test/integration/component/test_nuage_vsp.py

```diff
@@ -1,334 +0,0 @@
-                "displayname": "TestVM",
-                "username": "root",
-                "password": "password",
-                "ssh_port": 22,
-                "hypervisor": 'KVM',
-                "privateport": 22,
-                "publicport": 22,
-                "protocol": 'TCP',
-            },
-            "nuage_vsp_device": {
-                "hostname": '172.31.222.162',
-                "username": 'cloudstackuser1',
-                "password": 'cloudstackuser1',
-                "port": '8443',
-                "apiversion": 'v3_2',
-                "retrycount": '4',
-                "retryinterval": '60'
-            },
-            # services supported by Nuage for isolated networks.
-            "network_offering": {
-                "name": 'nuage_marvin',
-                "displaytext": 'nuage_marvin',
-                "guestiptype": 'Isolated',
-                "supportedservices":
-                'Dhcp,SourceNat,Connectivity,StaticNat,UserData,Firewall',
-                "traffictype": 'GUEST',
-                "availability": 'Optional',
-                "serviceProviderList": {
-                    "UserData": 'VirtualRouter',
-                    "Dhcp": 'NuageVsp',
-                    "Connectivity": 'NuageVsp',
-                    "StaticNat": 'NuageVsp',
-                    "SourceNat": 'NuageVsp',
-                    "Firewall": 'NuageVsp'
-                },
-                "serviceCapabilityList": {
-                    "SourceNat": {"SupportedSourceNatTypes": "perzone"},
-                }
-            },
-            "network": {
-                "name": "nuage",
-                "displaytext": "nuage",
-            },
-            "ostype": 'CentOS 5.5 (64-bit)',
-            "sleep": 60,
-            "timeout": 10
-        }
-
-
-class TestNuageVsp(cloudstackTestCase):
-
-    @classmethod
-    def setUpClass(cls):
-        print "In setup class"
-        cls._cleanup = []
-        cls.testClient = super(TestNuageVsp, cls).getClsTestClient()
-        cls.api_client = cls.testClient.getApiClient()
-
-        cls.services = Services().services
-        # Get Zone, Domain and templates
-        cls.domain = get_domain(cls.api_client)
-        cls.zone = get_zone(cls.api_client, cls.testClient.getZoneForTests())
-        cls.template = get_template(
-            cls.api_client,
-            cls.zone.id,
-            cls.services["ostype"]
-        )
-        # nuage vsp device brings the Nuage virtual service platform into play
-        cls.nuage_services = cls.services["nuage_vsp_device"]
-        try:
-
-            resp = listPhysicalNetworks.listPhysicalNetworksCmd()
-            print "in cls.setupClass- resp: %s" % resp
-            resp.zoneid = cls.zone.id
-            physical_networks = cls.api_client.listPhysicalNetworks(resp)
-            for pn in physical_networks:
-                if pn.isolationmethods=='VSP':
-                    physical_network = pn
```

Comment:
I would say come out of the loop as and when you find a physical network with isolation method as "VSP" unless the intention is to use multiple physical networks.